Policies/github-terms-of-service.md

```diff
@@ -36,279 +38,2585 @@ Thank you for using GitHub! We're happy you're here. Please read this Terms of S
 | [Q. Changes to these Terms of Service](#q-changes-to-these-terms) | We may modify this agreement, but we will give you 30 days' notice of changes that affect your rights. |
 | [R. Miscellaneous](#r-miscellaneous) | Please see this section for legal details including our choice of law. |
 
-### The GitHub Terms of Service
-Effective date: November 13, 2019
+</details><BR>
 
+---
+
+# The GitHub Terms of Service
+<sup><sup><sub><sub><sub>🔱</sub></sub></sub></sup></sup> Effective date: November 13, 2019
+
+<BR>
+<TABLE BORDER="1px"><TR><TD>
+
+## Important Preliminaries
+
+#### Conditional-text devices
+  <table>
+  <tr><td valign="top"><sup><sup><sub><sub><sub>🔱</sub></sub></sub></sup></sup> For each<br> 
+        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<i>one&#8209;row,&nbsp;three&#8209;cell</i>&nbsp;table<br>
```

Comment:
Could you not to use unprintable characters for alignment?

Reply:
Hello @KOLANICH, not precisely sure to what you are referring, but I think you might be referring to the use of tables where the borders of the tables are displayed.

The typography and visualisations are very much in-development, new, and untested. The more I've worked on these terms and conditions, the more I've developed new visualisations. In respect of the table, I think that was probably the only way I thought was possible for the kind of alignment I wanted. If I had known of another way where the alignment didn't involve displaying extra things, then I would have gone for it.

Looking at what you have highlighted, I probably wanted to split the relatively long sentence, into three visually-separate chunks, for the facilitating of easier and quicker comprehension. For that, it seems that I chose to use a three-cell, single-row table, with each cell containing a different part of the sentence.

Am planning on returning to this work soon. Want to complete the work now. Plan then to use it as an example to others exampling the ways in which the present state of online terms and conditions can be much improved.

Thanks.